packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx

```diff
@@ -25,8 +24,8 @@ export const DashboardList: FC<Props> = ({ resourceItemId, projectUuid }) => {
                     {relatedDashboards.map(({ uuid, name }) => (
                         <List.Item key={uuid}>
                             <Anchor
-                                component={Link}
-                                to={`/projects/${projectUuid}/dashboards/${uuid}/view`}
+                                href={`${window.location.origin}/projects/${projectUuid}/dashboards/${uuid}/view/`}
```

Comment:
changing from Link (react-router-dom) removes SPA navigation functionality (refreshes the page) my understanding is that `<Anchor component={Link}` supports opening to a new tab. maybe this bug has something to do something with `e.preventDefault()`?

Reply:
Yes, it boots up / refreshes the app again for a new tab. Its not above `e.preventDefault()`, I tested by removing just that and it doesn't open in new tab.,
And as far as I know, `Link` from react-router is only for internal routing, here we are trying to open in a new tab, hence going out. 
Correct me if I'm wrong please

Reply:
ok! current changes look good then!

Reply:
Thank you for your contribution @muhsinkamil! ⚡ 🙌🏼